packages/monaco/src/vim-keybindings.ts

```diff
@@ -0,0 +1,38 @@
+import { type VimMode } from 'monaco-vim';
+
+export default function registerKeyBindings(vim: typeof VimMode) {
```

Comment:
I think we prefer named exports instead of default exports

Reply:
Me too, updating as per your pr notes